server/migrations/1605755216563_initial-schema.js

```diff
@@ -1,12 +1,15 @@
 /* eslint-disable no-restricted-syntax, no-await-in-loop */
-const { schema } = require('../db/schema.js');
-const { dbClient } = require('../db/db.js');
+const { dbClient, schema } = require('../db');
 
 exports.up = async () => {
-  for (const schemaItem of schema) {
+  for (const schemaItem of schema.relationalTables) {
+    await dbClient.runRawQuery(schemaItem.definition);
+  }
+  for (const schemaItem of schema.documentTables) {
     await dbClient.createDocumentTableIfNotExists(schemaItem.collection);
     for (const index of schemaItem.indexes) {
-      dbClient.createIndexIfNotExists(schemaItem.collection, index);
+      await dbClient.createIndexIfNotExists(schemaItem.collection, index);
     }
   }
+  await dbClient.reload(); // Required for subsequent migrations to see all tables
```

Comment:
I think this might have been missing from some of my previous work. If you have two migrations that get applied at the same time, will the db need to be reloaded before the second migration can see a table made by the first one?